src/ManageCourses.Api/Services/EnrichmentService.cs

```diff
@@ -178,24 +178,30 @@ public bool PublishInstitutionEnrichment(string instCode, string email)
         /// <param name="instCode">institution code for the enrichment</param>
         /// <param name="ucasCourseCode">can be null</param>
         /// <param name="email">email of the user</param>
+        /// <param name="publishableOnly">if there is a draft enrichment then get the ucas contact details</param>
         /// <returns>An enrichment object with the enrichment data (if found). Nul if not found</returns>
-        public UcasCourseEnrichmentGetModel GetCourseEnrichment(string instCode, string ucasCourseCode, string email)
+        public UcasCourseEnrichmentGetModel GetCourseEnrichment(string instCode, string ucasCourseCode, string email, bool publishableOnly)
```

Comment:
Can we make this a private and have two public for the false and true

Reply:
This creates a further refactor issue so we'll card this up to be done at a later date